sdk/communication/communication-common/src/entraTokenCredential.ts

```diff
@@ -81,14 +81,25 @@ export class EntraTokenCredential implements AcsTokenCredential {
 
   private async getTokenInternal(options?: CommunicationGetTokenOptions): Promise<AccessToken> {
     const getTokenOptions = options?.abortSignal ? { abortSignal: options.abortSignal } : undefined;
-    const token = await this.options.tokenCredential.getToken(this.options.scopes, getTokenOptions);
+    const token = await this.options.tokenCredential.getToken(
+      this.options.scopes
+        ? this.options.scopes
+        : ["https://communication.azure.com/clients/.default"],
+      getTokenOptions,
+    );
+    const currentDateTime = new Date(Date.now());
```

Comment:
I think it's safe to just use the empty constructor to get a date for "now".
```suggestion
    const currentDateTime = new Date();
```